boards/arm/nrf52810_pca10040/nrf52810_pca10040.dts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2017 Shawn Nock <shawn@monadnock.ca>
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52810.dtsi>
+
+/ {
+	model = "Nordic PCA10040 Dev Kit";
+	compatible = "nordic,pca10040-dk", "nordic,nrf52810-qfaa",
+		     "nordic,nrf52810";
+
+	chosen {
+		zephyr,console = &uart0;
```

Comment:
nrf52810 does not have an UART0, only UARTE0

Reply:
While @carlescufi comment is correct, if its the case that UARTE0 can only be the 'E' form, than the compat in the SOC dts should just be 'nordic,nrf-uarte' and we don't need to update the compat in the board.dts

Reply:
It's already the case that compat in the SoC dts is only nrf-uarte

Reply:
then i suppose we are ok.